### PR TITLE
login/register ejs

### PR DESCRIPTION
**Pull request by charlottehauke**
Fixed login html alignment and added register ejs

### CODE REVIEW

File: Project Code/views/pages/login.ejs [hunk 1]
```diff
@@ -1,19 +1,30 @@
...
-            <label for="name" >Username: </label>
-            <input required type="text"  id="username" name="username" />
-        </div>
-        <div>
-            <label for="password">Password:  </label>
-            <input required type="text" id="password" name="password" />
-        </div>
-            <input style="margin-top:60px; margin-bottom: 20px;"type="submit" value="Login" />
+                <label for="name">Username: </label>
+                <input required type="text" id="username" name="username" />
```
Comment (navanchauhan) [inline Project Code/views/pages/login.ejs:18]: Should we go for email instead of username?

File: Project Code/views/pages/register.ejs [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+            <div style="margin-bottom:20px;">
+                <label for="email">Email: </label>
+                <input required type="text" id="email" name="email" />
+            </div>
+
+            <div style="margin-bottom:20px">
+                <label for="username">Username: </label>
+                <input required type="text" id="username" name="username" />
+            </div>
```
Comment (navanchauhan) [inline Project Code/views/pages/register.ejs:36]: If we go for email only in the login form, we can drop this requirement. But if we want to keep the username, then this is good

File: Project Code/views/pages/login.ejs (hunk 1 above)
Comment (charlottehauke) [reply Project Code/views/pages/login.ejs:18]: Yeah email sounds good I'll change it right quick

**Review by navanchauhan — APPROVED** [review]
LGTM 🚀

**Pull request merged**